### DISCUSSION TITLE
What is the correct way to simulate a 2D vertical model with a hydrostatic gradient?

### DISCUSSION THREAD

**Discussion by pierre-jeanne** [pmgbergen/porepy#592] [Q&A]
Hello,
I am very new with porepy (I've worked a lot with TOUGH2) and now I am struggling with (I am sure) very simple things. I checked the tutorials, but I didn't find my answer.
 So, what are the steps to create a hydrostatic and geothermal gradient with porepy.
Thanks1

**Comment by IvarStefansson** [comment]
Hi,
We are currently working on improving the documentation, including a [tutorial ](https://github.com/pmgbergen/porepy/blob/incompressible_flow_tutorial/tutorials/incompressible_flow_model.ipynb )on incompressible flow showing how to include gravity effects. This has not been included in develop, so please refer to the incompressible-flow-tutorial branch, keeping in mind that this is work in progress.
Ivar

**Reply by pierre-jeanne** [reply]
Thank you, Ivar!
I will have a look at it.
P.
